src/containers/token-bridge/components/explainers.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
I tried sending an approve token transaction I got a transaction failed (because of a bug with metamask, not the app).
Tried again, it worked.
Next, I went ahead to relay the tokens. The wallet predicted failure and when mined indeed the transaction failed: kovan.etherscan.io/tx/0x0b8122c3f9c69366aedc01cc274431390c2f1ffaeedd5424ba045b3518b13175.

The issue is that transcation failure doesn't seem to be handled properly now, in the first case it displayed the transfer successful page and in the second failure it just hung with a perma loading indicator.

Finally, I refreshed the page and tried again to relay the tokens but the wallet predicted failure so I cancelled. The UI displayed the PNK successfully converted screen again.

Reply:
Any ideas why it reverted? I'm investigating that now.
I find this odd because I tested for failure conditions as well and it should at least give a proper feedback.